src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs

```diff
@@ -371,6 +390,52 @@ public static void Verify_ManagedUnmanagedFunctionPointersDontMatch()
         extern static string CallManagedMethod(UserDataClass d, delegate* unmanaged[Cdecl]<void> fptr);
     }
 
+    class InheritanceBase
+    {
+        private static string OnBase() => nameof(OnBase);
+        private static string FieldOnBase = nameof(FieldOnBase);
+    }
+
+    class InheritanceDerived : InheritanceBase
+    {
+        private static string OnDerived() => nameof(OnDerived);
+        private static string FieldOnDerived = nameof(FieldOnDerived);
+    }
```

Comment:
Should we also test calling a virtual (abstract?) instance method of a base class on a derived object?  (ie: unsafe accessor emits a `callvirt`, not a `call`)

Should we test that looking for a virtual (non-virtual?) instance method defined in a base class but using a derived class as the unsafe accessor target will work?